frontend/src/app/BackButton.jsx

```diff
@@ -0,0 +1,12 @@
+import { useNavigate } from "react-router-dom";
+import { Button } from "@chakra-ui/react";
+
+function BackButton() {
+  const navigate = useNavigate();
+  const handleBackClick = () => {
+    navigate(-1);
```

Comment:
Is it necessary to rename the `useNavigate` to `navigate`?

Reply:
useNavigate() is a react Hook and cannot be called directly within other functions. When I try, it throws this error:
![IMG_8204](https://github.com/user-attachments/assets/9e93fe3b-8b64-457b-b348-304a2d26e52f)